pkg/workloads/cortex/downloader/download.py

```diff
@@ -29,10 +29,17 @@ def start(args):
     for download_arg in download:
         from_path = download_arg["from"]
         to_path = download_arg["to"]
+        item_name = download_arg.get("item_name", "")
```

Comment:
more preferrable to do a .get("item_name") and then check if it `item_name is not None` instead of `item_name != ""`

Reply:
will leave as is, as discussed